Kernel/Syscalls/execve.cpp

```diff
@@ -618,6 +625,10 @@ ErrorOr<void> Process::do_exec(NonnullLockRefPtr<OpenFileDescription> main_progr
     auto credentials = this->credentials();
     auto auxv = generate_auxiliary_vector(load_result.load_base, load_result.entry_eip, credentials->uid(), credentials->euid(), credentials->gid(), credentials->egid(), path->view(), main_program_fd_allocation);
 
+    // FIXME: How much stack space does process startup need?
```

Comment:
I don't understand this fixme so I left it as is. I'm inclined to remove it but I am not sure about it.